modules/service/src/main/scala/lucuma/odb/graphql/mapping/AtomRecordMapping.scala

```diff
@@ -13,13 +13,13 @@ import lucuma.odb.graphql.binding.TimestampBinding
 import lucuma.odb.graphql.predicate.Predicates
 
 import table.AtomRecordTable
-import table.StepRecordTable
+import table.StepRecordView
```

Comment:
This is unrelated as well, but `StepRecordTable` should have been (re)named `StepRecordView` since it refers to columns in the view.